security/jpa/src/test/resources/import.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO md5_entity (id, username, password, role) VALUES (1, 'admin', '21232f297a57a5a743894a0e4a801fc3', 'admin');
+INSERT INTO md5_entity (id, username, password, role) VALUES (2, 'user','ee11cbb19052e40b07aac0ca060c23ee', 'user');
```

Comment:
Well, you can't say I didn't warn you that if you request my review I'll want also scenario where you create the user :-)

What you test is fine, but please, test whole flow, users are not a static data and there is a good change that application will create these users, so I think you should have a smoke test that persist new user and then uses it. Nothing complicated.

Choice is yours, but for example (do what you want!) I would just replace one of imported users, like create 'user' with 'admin' while 'admin' is important in this SQL.

Point behind this request: test real-world scenarios.

Reply:
It might also be of interested to delete user, but we can leave it for now as it's a dev preview. Your call.

Reply:
Correction: you didn't request my review, it was Rostislav :-D